cmd2/cmd2.py

```diff
@@ -703,6 +703,9 @@ def ppaged(self, msg: str, end: str='\n', chop: bool=False) -> None:
                         else:
                             break
                     self.pipe_proc = None
+                elif self.redirecting and self.colors.lower() in (constants.COLORS_NEVER.lower(),
```

Comment:
I don’t think you need the elif.  I think you can just change the callnin the else below to one to self.decooorized_write.

Reply:
the elif should handle users with `colors = Always`

Reply:
oh nevermind, that code handles it. updating. also adjusted the docs.